### ISSUE TITLE
The information text given to the user uses the old command format, leading to confusion

### ISSUE THREAD

**Issue by XenHat** [OpenCollar/opencollar#604]
I put the `OpenCollar 4.0 BETA - Part III` on while using a viewer not capable of using the RLV API and I apparently can't disable the collar's RLV checks as instructed (with chat command):

```
[22:17] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[22:17] OpenCollar 4.0 BETA - Part III: @clear
[22:17] OpenCollar 4.0 BETA - Part III: Beep: 20150726.1
[...]
[22:18] OpenCollar 4.0 BETA - Part III: 

If your viewer doesn't support RLV, you can stop the "@versionnew" message by switching RLV off in your collar's RLV menu or by typing: xerlvoff

[22:19] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[...]
[22:20] Xenhat: . . .
[22:20] Xenhat: Apparently this new collar doesn't listen on channel 1
[22:20] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[...]
[22:21] Xenhat: xerlvoff
[22:21] Xenhat: o.O
[22:21] Xenhat: I CANT STOP IT
[22:21] Xenhat: LOL
[22:21] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[22:22] Xenhat: xerlvoff
[22:22] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[...]
[22:26] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[22:26] Xenhat: xerlvoff
[22:27] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[22:29] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[22:30] OpenCollar 4.0 BETA - Part III: @versionnew=293847
[...]
[22:35] OpenCollar 4.0 BETA - Part III: 

If your viewer doesn't support RLV, you can stop the "@versionnew" message by switching RLV off in your collar's RLV menu or by typing: xerlvoff

[22:35] Xenhat: xerlvoff
[22:35] OpenCollar 4.0 BETA - Part III: @versionnew=293847
```

Hmmm... This collar seems to be pretty stubborn. :smiley_cat: 

P.S. The Menu command works fine.

Possibly related to #540

**Comment by GarvinTwine** [comment]
related to #572 somehow forgot the change the info text to the new command:
<prefix> rlv off instead of <prefix>rlvoff
so the bug is the wrong info text with the old command.

**Comment by XenHat** [comment]
Oh right, I remember seeing this earlier. Mentally noted.

**Comment by GarvinTwine** [comment]
fixed in #605

**Comment by XenHat** [comment]
Tested for completion's sake. :+1:
Thanks!
